### DISCUSSION TITLE
Gay movies

### DISCUSSION THREAD

**Discussion by Markd020** [adultmm/AdultMediaManager#5] [Ideas]
The list is long, but seems to be targeted to a straight community. Is it possible to add gay male responses to the scraping results?

**Comment by adultmm** [comment]
Hi Markd020,

For networks that have gay contents my scraper supports them. You can request any sites you want, just [create an issue](https://github.com/adultmm/AdultMediaManager/issues). I implement features and scrapers in popularity order. 

Have a nice day,
amm
